### ISSUE TITLE
Only "has your issue already been reported" should be shown initially

### ISSUE THREAD

**Issue by wilzbach** [CyberShadow/bugzilla-meta#3]
I really like the way Bugzilla solved this:

![image](https://user-images.githubusercontent.com/4370550/37940771-c5d96dcc-316a-11e8-88a0-bb36908919b2.png)

Also I like that the keywords I entered are copied to "Summary" if the issue isn't found.

**Comment by CyberShadow** [comment]
I think issues.dlang.org currently already has something like this, right?

**Comment by CyberShadow** [comment]
Oh, maybe it doesn't...

**Comment by wilzbach** [comment]
Yeah it does, but it does so when you are already at the input mask and type in the bug's summary:

![image](https://user-images.githubusercontent.com/4370550/37940972-d0511f06-316b-11e8-95c6-70f02ef83410.png)

(yes, it's one of the few features I like about issues.dlang.org)
